lib/vanagon/driver.rb

```diff
@@ -31,11 +31,12 @@ def initialize(platform, project, options = { :configdir => nil, :target => nil,
       engine = 'base' if target
       # Hardware has explicit teardown to unlock the node
       engine = 'hardware' if @platform.build_hosts
+      engine = 'docker' if @platform.docker_image
```

Comment:
This isnt super realted. But make sense if you had a docker_image that it should use docker.